testing/test_pluginmanager.py

```diff
@@ -217,7 +217,13 @@ def he_method1(self, arg):
     pm.register(Plugin1())
 
     he_method1 = pm.hook.he_method1
-    he_method1.call_historic(proc=callback, kwargs=dict(arg=1))
+
+    # TODO: remove once ``proc`` is removed
+    if callback:
```

Comment:
For completeness we should also test passing `result_callback`. I suggest changing this test to use `result_callback` and create a new, simpler test, just for the deprecation.

Reply:
Sure thing!